designs/nmi_dbus_interface.md

```diff
@@ -0,0 +1,92 @@
+# Design proposal for issuing NMI on servers that use OpenBMC
```

Comment:
We haven't been pulling any designs into our PR's. Any reason you did it here?

Reply:
I was first hesitant to do it. I can remove it if not required